providers/summary/clearlydefined.js

```diff
@@ -102,6 +105,16 @@ class ClearlyDescribedSummarizer {
   addNuGetData(result, data) {
     setIfValue(result, 'described.releaseDate', extractDate(data.releaseDate))
     setIfValue(result, 'licensed.declared', extractLicenseFromLicenseUrl(get(data, 'manifest.licenseUrl')))
+    const packageEntries = get(data, 'manifest.packageEntries')
+    if (packageEntries) {
```

Comment:
Bit of formatting, naming, and code ordering suggestions.
```suggestion
    if (!packageEntries) return
    const newDefinition = cloneDeep(result)
    newDefinition.files = packageEntries.map(file => { return { path: decodeURIComponent(file.fullName) } })
    mergeDefinitions(result, newDefinition)
  }
```